pkg/storage/testdata/mvcc_histories/ambiguous_writes

```diff
@@ -16,7 +16,7 @@ with t=A k=a
   put v=first
 ----
 >> put v=first t=A k=a
-put: lock acquisition = {a id=00000001 key="a" iso=Serializable pri=0.00000000 epo=0 ts=11.000000000,0 min=0,0 seq=0 Replicated Intent []}
+put: lock acquisition = {span=a id=00000001 key="a" iso=Serializable pri=0.00000000 epo=0 ts=11.000000000,0 min=0,0 seq=0 durability=Replicated strength=Intent ignored=[]}
```

Comment:
are we breaking an API here by adding the `span=` etc. to the output?

Reply:
I was using "API" loosely to mean the log format. (My perspective is that if a test is checking exact log output then there's an expectation that this output stays static.) Agree that nothing else should be affected.